packages/twenty-front/src/modules/favorites/states/selectors/favoriteViewsWithMinimalDataSelector.ts

```diff
@@ -0,0 +1,18 @@
+import { prefetchViewsState } from '@/prefetch/states/prefetchViewsState';
+import { View } from '@/views/types/View';
+import { selector } from 'recoil';
+
+export const favoriteViewsWithMinimalDataSelector = selector<
```

Comment:
selector designed for favorites that need to have minimal info on views (to avoid re-renders too)

Reply:
Could we use viewsForMenuItem or something that tells us what the props curation is for ?